Pod/Classes/WPMediaCollectionDataSource.h

```diff
@@ -317,5 +317,14 @@ typedef int32_t WPMediaRequestID;
  */
 - (BOOL)ascendingOrdering;
 
+@optional
+
+/**
+ *  Tells the Data Source that the search string has been changed
+ *
+ *  @param searchText the new search text
+ */
+- (void)searchTextDidChange:(nullable NSString *)searchText;
```

Comment:
I'm not a fan of this method name in the data source I'm more inclined to something like `filterByTerm` or `searchFor` this looks too associated to the delegate callback of the searchBar.